setup.cfg

```diff
@@ -41,7 +41,8 @@ exclude =
 #   B3??: Python 3 compatibility warnings
 #   B006: use of mutable defaults in function signatures
 #   B007: Loop control variable not used within the loop body.
-ignore = E251,E128,F401,W601,E701,W503,W504,E704,E402,B3,B006,B007
+#   B011: Don't use assert False
```

Comment:
What does flake8 want you to use???

Reply:
`Do not call assert False since python -O removes these calls. Instead callers should raise AssertionError().`